src/helpers/URLHelper.js

```diff
@@ -0,0 +1,10 @@
+var uri = require("URIjs");
+
+export function updateApplicationUrlState(url) {
```

Comment:
So that if someone hits the url without a query attached I can check in localStorage for what their last query was and run it. Request from JD and JM until we get a UX direction around saved searches.